### PR TITLE
MOSIP-23441 Corrected Readme file.

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

**Review by loganathan-sekaran — APPROVED** [review]

**Review by loganathan-sekaran — APPROVED** [review]

File: data-archive/archive-jobs/mosip_credential/README.md [hunk 1]
```diff
@@ -1,38 +1,53 @@
...
+the required status codes are added to the configuration in [mosip_archive_credential.ini](mosip_archive_credential.ini).
+For Example:- If today Date is 01-Sept-2022 then if
+```credential.status.time.map=ISSUED:2```.
+Then it will archive records 2 days before that is before 31-Aug-2022.
+
+ Set the status code to archive the data as below:
+ 
  ```credential.endstatus.list=STORED,printing```
+
 Above property will archive the mosip_credential table data based on the status code.
```
Comment (vishwa-vyom) [inline data-archive/archive-jobs/mosip_credential/README.md]: Lets additionally say that all the recods matching status in the endstatus list will be archived without any consideration on created dates.

File: data-archive/archive-jobs/mosip_credential/README.md [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
 
 ```credential.status.time.map=ISSUED:2,FAILED:7```
-Above property will archive the mosip_credential table data based on the status code and number of days.
```
Comment (vishwa-vyom) [inline data-archive/archive-jobs/mosip_credential/README.md:35]: Lets also tell what will be the behaviour if the same status is available in both credential.status.time.map and credential.endstatus.list

File: data-archive/archive-jobs/mosip_credential/README.md [hunk 3]
```diff
@@ -1,38 +1,53 @@
...
+The Jobs will archive the data from the **credential_transactions** table in mosip_credential db to the table in archive database.
 
 ## Pre-requisite steps:
 
 1. If we don't have mosip_archive DB than go ahead and continue with the [db_scripts](../../../data-archive/db_scripts)
 2. Once we have the mosip_archive DB ready we need to continue with the archive_jobs.
 3. Install python3.9 virtual env.
 4. Switch to virtual env.
-5. Install required psycopg dependency. `sudo yum install python3-psycopg
+5. Install required psycopg dependency. `sudo yum install python3-psycopg.
```
Comment (vishwa-vyom) [inline data-archive/archive-jobs/mosip_credential/README.md:11]: Lets say "Below are the 2 different ways of running the the archival script.
1. Archive jobs based on status code (Recommended)
2. Archive jobs based on no of days after the records were created (Old script)

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]
Also I am not reviewing the code changes, hope Loga will check on those.

File: data-archive/archive-jobs/mosip_credential/README.md (hunk 1 above)
Comment (kameshsr) [reply data-archive/archive-jobs/mosip_credential/README.md]: Done.

File: data-archive/archive-jobs/mosip_credential/README.md (hunk 2 above)
Comment (kameshsr) [reply data-archive/archive-jobs/mosip_credential/README.md:35]: Done.

File: data-archive/archive-jobs/mosip_credential/README.md (hunk 3 above)
Comment (kameshsr) [reply data-archive/archive-jobs/mosip_credential/README.md:11]: DOne.

**Review by vishwa-vyom — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**